src/main/java/com/github/valfirst/jbehave/junit/monitoring/JUnitReportingRunner.java

```diff
@@ -40,6 +41,9 @@ public JUnitReportingRunner(Class<? extends ConfigurableEmbedder> testClass)
 		configuredEmbedder = configurableEmbedder.configuredEmbedder();
 		configuration = configuredEmbedder.configuration();
 
+		String various = "Various";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "various" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.oss.wandera.net/coding_rules#rule_key=squid%3AS1481)